config/webpack.dev.config.js

```diff
@@ -64,9 +64,13 @@ module.exports = Merge.smart(commonConfig, {
       // bundles. This makes the javascript bundles ginormous and defeats caching so we will use the
       // file-loader instead to copy the files directly to the output directory.
       {
-        test: /\.(woff2?|ttf|svg|eot)(\?v=\d+\.\d+\.\d+)?$/,
+        test: /\.(woff2?|ttf|eot)(\?v=\d+\.\d+\.\d+)?$/,
         loader: 'file-loader',
       },
+      {
+        test: /.svg$/,
+        use: ['@svgr/webpack', 'url-loader'],
```

Comment:
Curious what this is all about. :)

Reply:
This is the webpack loader for `svg` files.